alarm/PolicyManager2.js

```diff
@@ -64,11 +67,24 @@ class PolicyManager2 {
         log.info("got policy enforcement event:" + event.action + ":" + event.policy.pid)
 
         if (event.action && event.action == 'enforce') {
-          this.enforce(event.policy)
+          try {
```

Comment:
try catch doesn't work here, have to be inside async scope

Reply:
outside of async scope, "try" will not catch the errors from promise

Reply:
and better change this section code to async/await, i'm slowly changing all callback/promise to async/await.